tracing/src/span.rs

```diff
@@ -467,9 +470,31 @@ impl Span {
         }
     }
 
+    /// Returns a handle to the span [considered by the `Subscriber`] to be the
+    /// currrent span.
+    ///
+    /// If the subscriber indicates that it does not track the current span, or
+    /// that the thread from which this function is called is not currently
```

Comment:
The "thread" business you're referring to is the thing you described offline to me, correct? The one in which spans try _really_ hard to not to be moved?

Reply:
No, it's just that the expectation is that typically, a current span is expected to be tracked on a per-thread basis --- one thread might enter a given span, and another thread enters a totally disparate thread.